Yamaha/Views/ProductCell.swift

```diff
@@ -13,19 +13,13 @@ class ProductCell: UICollectionViewCell {
   static let identifier = "ProductCell"
   
   // Variables
-  private(set) var product: Product!
+  private(set) var displayedProduct: Product!
+  private(set) var displayedLstAll: LstAll!
+  private(set) var displayedBolo: Bolo!
```

Comment:
you're not assigning anything from this. setProduct and setLstAll does not put value to displayedProduct and displayedLstAll
- either remove this or use this